NorthstarDedicatedTest/scriptUtility.cpp

```diff
@@ -0,0 +1,28 @@
+#include "pch.h"
+#include "scriptUtility.h"
+#include "squirrel.h"
+
+SQRESULT ServerSq_StringToAsset(void* sqvm)
```

Comment:
should be SQ_ not Sq_ to be inline with other squirrel funcs
could also template these funcs rather than having dupe ones for client/server

Reply:
are you sure with other functions that have client and server versions we also had ServerSq and ClientSq as prefix
![image](https://user-images.githubusercontent.com/11448698/177016025-a3109348-dd38-42a3-a87b-0817d5df2e29.png)
but im making it a template anyway so it shouldn't matter